### DISCUSSION TITLE
Request approved for GitHub Education Benefits but haven't gained access to it even though it has been more than 5 days.

### DISCUSSION THREAD

**Discussion by aryankhare2110** [community/community#144248] [GitHub Education]
### Hi everyone,

I have been approved for the GitHub Education Benefits but have not received them yet. It has been over 5 days, and I am unsure how to proceed. Can someone help?
![Screenshot 2024-11-12 100259](https://github.com/user-attachments/assets/534c789a-40a6-4847-81ff-c76cec59eecf)

**Comment by davevad93** [comment]
Hi @aryankhare2110 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @aryankhare2110 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
